spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/channel/ChannelMessageStoreQueryProvider.java

```diff
@@ -125,4 +126,15 @@ default String getDeleteMessageGroupQuery() {
 	 */
 	String getPriorityPollFromGroupQuery();
 
+	/**
+	 * Specifies if the returned query for polling is using a single statement (e.g. DELETE ... RETURNING) to
+	 * retrieve and delete the message from the channel store.
+	 *
+	 * @return false if a select and delete statement is required, true if otherwise.
+	 * @since 6.2
+	 */
+	default boolean isUsingSingleStatementForPoll() {
```

Comment:
No need in `Using`. I guess just `isSingleStatementForPoll` is enough to be clear.
Please, use these rules for method Javadocs: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting

Reply:
done.

Reply:
You know, no need to say `Done` on every single my comment in review.
If you agree with me, there is just enough to push such a fix and I'll see the change as is.
Feel free to continue discussion otherwise.